File: FuNWallet.Client/FuNWallet.Client/Models/Transaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FuNWallet.Client.Models
{
    public class Transaction
    {
        public string RecipientID { get; set; }
        public double Amount { get; set; }
        public string Message { get; set; }
    }
}

File: src/main/java/AppControl/Transaction.java
package AppControl;

public class Transaction {
    private int senderID;
    private int receiverID;
    private int amount;

    public Transaction(int senderID, int receiverID, int amount) {
        this.senderID = senderID;
        this.receiverID = receiverID;
        this.amount = amount;
    }
}
